src/MoBi.UI/Services/ApplicationStartup.cs

```diff
@@ -218,6 +221,7 @@ public void InitializeForStartup(Action<IContainer> registrationAction)
 
          container.AddRegister(x => x.FromType<PresenterRegister>());
          container.AddRegister(x => x.FromType<UIRegister>());
+         container.AddRegister(x => x.FromType<InfrastructureRegister>());
```

Comment:
Had to move this to occur before `configureLogger`. That was necessary to really re-use the snapshot import presenter and view from PK-Sim. The logging mechanism is how the snapshot importer messaging works.